src/bindings/python/docs/test_examples.md

```diff
@@ -8,6 +8,10 @@ Install the specific requirements file for testing:
 python -m pip install -r openvino/src/bindings/python/requirements_test.txt
 ```
 
+Make sure that Python libraries are added to the user environment variables: 
+```
+export PYTHONPATH=PYTHONPATH:<openvino_repo>/bin/intel64/Release/python_api/python3.7
+```
 ### Run OpenVINO™ Python API tests
 *For simplicity, all of these commands require to navigate to the main Python API folder first:*
```

Comment:
How about we add "the user" or "you"? Like:
"require the user to navigate to the"
Just a thought. You decide if it sounds good ;)

Reply:
I'd prefer to keep my version in this case  :)